tests/test_perseptual.py

```diff
@@ -71,10 +72,10 @@ def test_content_loss_supports_custom_extractor(prediction: torch.Tensor, target
 @pytest.mark.parametrize(
     "prediction,target,expectation,value",
     [
-        (torch.rand(4, 3, 128, 128, 2), torch.rand(4, 3, 128, 128, 2), pytest.raises(AssertionError), None),
-        (torch.randn(4, 3, 128, 128), torch.randn(4, 3, 128, 128), pytest.raises(AssertionError), None),
-        (torch.zeros(4, 3, 128, 128), torch.zeros(4, 3, 128, 128), raise_nothing(), 0.0),
-        (torch.ones(4, 3, 128, 128), torch.ones(4, 3, 128, 128), raise_nothing(), 0.0),
+        (torch.rand(4, 3, 96, 96, 2), torch.rand(4, 3, 96, 96, 2), pytest.raises(AssertionError), None),
+        (torch.randn(4, 3, 96, 96), torch.randn(4, 3, 96, 96), pytest.raises(AssertionError), None),
+        (torch.zeros(4, 3, 96, 96), torch.zeros(4, 3, 96, 96), raise_nothing(), 0.0),
+        (torch.ones(4, 3, 96, 96), torch.ones(4, 3, 96, 96), raise_nothing(), 0.0),
```

Comment:
I think you can also reduce number of elements in the batch here to 2 without the loss of generality

Reply:
Done